pkg/skaffold/deploy/util.go

```diff
@@ -133,6 +136,23 @@ func dumpToFile(renderedManifests string, filepath string) error {
 	return err
 }
 
+// Returns relative path pointing to the GCS temp dir
+func downloadManifestsFromGCS(manifests []string) (string, error) {
+	if err := os.MkdirAll(manifestTmpDir, os.ModePerm); err != nil {
```

Comment:
Can you please use create a `manifestTmpDir` in this function instead of  a variable using 
```
manifestTmpDir := ioutil.TempDir("", manifestsStagingFolder)
```

Reply:
See the comment above.